cloud/jenkins/psmdb_operator_gke_alpha.groovy

```diff
@@ -0,0 +1,313 @@
+void CreateCluster(String CLUSTER_PREFIX) {
```

Comment:
CLUSTER_PREFIX is used as suffix, so I propose to rename it everywhere

Reply:
https://github.com/Percona-Lab/jenkins-pipelines/pull/1009/commits/5e753a5a785f2f85ea63895d0d7eccf57bfcd61f